src/plugins/upf/pfcp.h

```diff
@@ -190,7 +190,7 @@ typedef struct {
 } pfcp_sdf_filter_t;
 
 #define PFCP_IE_APPLICATION_ID				24
-typedef u32 pfcp_application_id_t;
+typedef u8 * pfcp_application_id_t;
```

Comment:
The feature/upf was update to implement this (and your version is also wrong). This needs to be rebased to the current head of feature/upf